test/models/audio_file_test.rb

```diff
@@ -15,6 +15,16 @@
 require 'test_helper'
 
 class AudioFileTest < ActiveSupport::TestCase
+  class ProcessStub
+    def initialize(success, exit_status)
+      @success = success
+      @exit_status = exit_status
+    end
+
+    def success? = @success
```

Comment:
Couldn't we simplify this to:
```suggestion
    def initialize(exit_status)
      @exit_status = exit_status
    end

    def success? = @exit_status == 0
```